src/main.rs

```diff
@@ -49,7 +49,13 @@ fn calculate_safety_status(
     has_carrot: bool,
     friends_nearby: i32,
 ) -> bool {
-    todo!()
+    if wolves_nearby == true || day_time == false {
+        if has_carrot == true || friends_nearby > 3 {
+            return true;
+        }
+        return false;
+    }
+    return true;
```

Comment:
This is pretty good! But you can actually simplify this quite a lot by using boolean logic all on one line. It might look something like `a || (b && c)`.